lib/App/DuckPAN/Fathead.pm

```diff
@@ -26,29 +26,21 @@ sub _trigger_selected {
 		my $full_path = $dir->realpath;
 		$self->app->emit_and_exit(1, "Directory not found: $full_path") ;
 	}
+	my $file = $dir->child("output.txt");
+	unless ($file->exists){
+		my $full_path = $file->realpath;
+		$self->app->emit_and_exit(1, "No output.txt was not found in $full_path");
+	}
+	$self->output_txt($file);
 	return $dir;
 }
 
 has output_txt => (
 	is => 'rw',
```

Comment:
Why don't we make this `rwp`? Then we can use `$self->_set_output_txt(...)`.

I doubt anything else would need to change the `output_txt`?